x11-misc/revelation/revelation-0.5.5.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
8 work?

Reply:
With some work to migrate to the EAPI 8 version of xdg.